src/frontend/src/lib/components/receive/ReceiveButton.svelte

```diff
@@ -2,14 +2,10 @@
 	import { isBusy } from '$lib/derived/busy.derived';
 	import IconReceive from '$lib/components/icons/IconReceive.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
-	import { IconQRCodeScanner } from '@dfinity/gix-components';
-	import ButtonDualMode from '$lib/components/ui/ButtonDualMode.svelte';
-
-	export let compact = false;
+	import ButtonHero from '$lib/components/ui/ButtonHero.svelte';
 </script>
 
-<ButtonDualMode on:click disabled={$isBusy} alt={$i18n.receive.text.receive} {compact}>
```

Comment:
The component `ButtonDualMode` can be deleted?

Reply:
Indeed! But I will do it in another PR

Reply:
Done with PR #1833